benchmarks/search/radius_search.cpp

```diff
@@ -0,0 +1,53 @@
+#include <pcl/io/pcd_io.h>
+#include <pcl/search/organized.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <benchmark/benchmark.h>
+
+static void
+BM_OrganizedNeighborSearch(benchmark::State& state, const std::string& file)
+{
+  pcl::PointCloud<pcl::PointXYZ>::Ptr cloudIn(new pcl::PointCloud<pcl::PointXYZ>);
+  pcl::PCDReader reader;
+  reader.read(file, *cloudIn);
+
+  pcl::search::OrganizedNeighbor<pcl::PointXYZ> organizedNeighborSearch;
+
+  double radiusSearchTime = 0;
+
+  for (auto _ : state) {
+    organizedNeighborSearch.setInputCloud(cloudIn);
+
+    int randomIdx = rand() % cloudIn->size();
```

Comment:
Wouldn't it make sense to use a set of fixed indexes and radiies?
Not sure if these rands can make it difficult to compare etc?

Reply:
Yeah, that is a good point. Random indices would at least add more noise in the time measurements, so that more iterations are necessary to reach a stable result. Alternatively, we could simply use every point from the input cloud once as the search point, and a fixed radius for all searches, e.g. 0.1 or 0.05?